### PR TITLE
Add img props to infoCard component

### PR DESCRIPTION
**Pull request by diegoleone**
#### What problem is this solving?

Currently we do not support `fetchpriority` & `preload` props for images within the native infoCard component. Therefore, I am adding those props, to allow better management of those assets. The implementation follows the example used in the vtex.store-image native component.
https://github.com/vtex-apps/store-image/blob/master/react/Image.tsx

#### How to test it?

Visit the Dunnes store in this workspace: https://preloadimg--dunnesstorespreprod.myvtex.com/

Screenshot 2024-08-07 at 12 36 11

### CODE REVIEW

**Comment by beatrizmaselli** [comment]
@diegoleone thanks for your contribution! Could you please update the docs folder with this new prop for the infoCard and add a changelog for this change? Thanks ✨

File: react/components/InfoCard/index.js [hunk 1]
```diff
@@ -97,6 +97,8 @@ const InfoCard = ({
   htmlId,
   textMode,
   linkTarget,
+  fetchpriority,
+  preload
```
Comment (beatrizmaselli) [inline react/components/InfoCard/index.js:101]: Please add the comma here or run a `yarn lint --fix`

**Review by beatrizmaselli — APPROVED** [review]
LGTM

File: react/components/InfoCard/index.js [hunk 2]
```diff
@@ -290,6 +298,8 @@ MemoizedInfoCard.defaultProps = {
   textMode: textModeTypes.TEXT_MODE_HTML.value,
   linkTarget: '_self',
   callToActionLinkTarget: '_self',
+  fetchpriority: 'auto',
+  preload: '',
```
Comment (iago1501) [inline react/components/InfoCard/index.js:302]: Since this is a boolean, this prop should be false and not line in isFullModeStyle ''

**Review by iago1501 — CHANGES_REQUESTED** [review]
Just a type fix

File: react/components/InfoCard/index.js (hunk 2 above)
Comment (iago1501) [inline react/components/InfoCard/index.js:302]:
```suggestion
  preload: false,
```

**Comment by diegoleone** [comment]
@iago1501 Thank you. All done.

**Review by iago1501 — APPROVED** [review]
LGTM

**Pull request merged**